src/stores/notifications/SummarizedNotificationState.ts

```diff
@@ -25,6 +26,7 @@ import { NotificationState } from "./NotificationState";
  */
 export class SummarizedNotificationState extends NotificationState {
     private totalStatesWithUnread = 0;
+    private unreadRoomId: string;
```

Comment:
The `unreadRoomId` and typed `RoomNotificationState` below still need to be removed

Reply:
Sure